sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -15,6 +15,9 @@
 #include <sycl/ext/intel/experimental/esimd/detail/memory_intrin.hpp>
 #include <sycl/ext/intel/experimental/esimd/detail/util.hpp>
 
+extern SYCL_EXTERNAL int32_t __spirv_BuiltInGlobalHWThreadIDINTEL();
```

Comment:
Sorry I just merged a change that makes it so we should use a new macro, `__DPCPP_SYCL_EXTERNAL` inside SYCL headers in https://github.com/intel/llvm/commit/9912ac290a9ca6a63e777905d0b3ace465c3bad5, and if we don't a test will fail.

Do you mind rebasing against current intel/llvm HEAD and updating the macro?

Sorry for the trouble, bad timing.

Reply:
Such SPIRV intrinscis/operations must be added to https://github.com/intel/llvm/blob/sycl/sycl/include/CL/__spirv/spirv_ops.hpp